tests/data/translated_titles/conf.py

```diff
@@ -225,6 +225,10 @@
 # output / TRANSLATION[lang] / RSS_PATH / rss.xml
 # RSS_PATH = ""
 
+# Final location for the blog main Atom feed is:
+# output / TRANSLATION[lang] / ATOM_PATH / index.atom
+# ATOM_PATH = ""
```

Comment:
(FYI, this file does not need to be updated.)

Reply:
Should we leave this change anyway?

Reply:
Whatever.